spec/mailers/comment_mailer_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+describe CommentMailer do
+  describe "#notification_new_comment" do
+    let!(:comment) { FactoryGirl.create(:comment) }
+    let(:mail) { CommentMailer.notification_new_comment(comment.id) }
+    
+    it "renders the subject" do
+      expect(mail.subject).to eql('New comment from your post')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.